src/main/java/com/Configuration/RedisConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.Configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.session.data.redis.config.ConfigureRedisAction;
+
+@Configuration
```

Comment:
Maybe it would be better to group all these @Bean in the same @Configuration class?

Reply:
I will group them together.